notifications_utils/formatters.py

```diff
@@ -178,6 +186,14 @@ def escape_html(value):
     return value
 
 
+def unescape_url(value):
```

Comment:
I’d strongly consider using `unescape_strict` from this module, rather than re-implementing a variation of the same thing.